plugins/adguardhome/adguardhome

```diff
@@ -0,0 +1,180 @@
+#!/bin/sh
+
+set -e
+
+: << =cut
+
+=head1 NAME
+
+adguardhome - Monitor dns queries
+
+=head1 APPLICABLE SYSTEMS
+
+AdGuardHome instances
+
+=head1 CONFIGURATION
+
+On a low traffic AdGuardHome instance you may want to lower 
+querylog_size_memory parameter in AdGuardHome configuration, otherwise 
+there won't be enough data for munin graphs.
+
+ log         - Querylog file of AdGuardHome
+ interval    - Interval of munin cronjob in minutes
+
+ [adguardhome]
+ env.log /opt/AdGuardHome/data/querylog.json
+ env.log 5
+
+=head1 AUTHOR
+
+Copyright (C) 2020 Sebastian L. (https://momou.ch)
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+case $1 in
+
+    autoconf)
+        if [ -x /opt/AdGuardHome/AdGuardHome ]; then
+            if [ -z "$(command -v AdGuardHome)" ]; then
+                echo "no (AdGuardHome not found)"
+                exit 0
+            else
+                echo "yes"
+                exit 0
+            fi
+        else
+            echo "no (/opt/AdGuardHome/AdGuardHome not found)"
+            exit 0
+        fi
+        ;;
+
+   config)
+        echo "multigraph adguardhome_queries"
+        echo "graph_title AdGuardHome - DNS Queries"
+        echo "graph_info DNS Queries made over AdGuardHome"
+        echo "graph_args -Y -A -l 0"
+        echo "graph_category network"
+        echo "graph_vlabel queries"
+        echo "queries.label Queries"
+        echo "queries.info All queries"
+        echo "queries.min 0"
+        echo "blocked.label Blocked"
+        echo "blocked.info Blocked queries by filterlist"
+        echo "blocked.min 0"
+        echo "parental.label Parental"
+        echo "parental.info Blocked by parental filter"
+        echo "parental.min 0"
+        echo "safesearch.label Safe search"
+        echo "safesearch.info Filtered for safe searching"
+        echo "safesearch.min 0"
+        echo "safebrowsing.label Safe browsing"
+        echo "safebrowsing.info Filtered for safe browsing"
+        echo "safebrowsing.min 0"
+        echo "multigraph adguardhome_avgproc"
+        echo "multigraph adguardhome_query_types"
+        echo "graph_title AdGuardHome - Query types"
+        echo "graph_info Query types made by AdGuardHome"
+        echo "graph_vlabel query types"
+        echo "graph_args -Y -A -l 0"
+        echo "graph_category network"
+        for KEY in A AAAA ANY CNAME MX NS NXDOMAIN PTR SIG SOA SRV TXT; do
```

Comment:
Since you are repeating this list of keys a few times, I would suggest to assign this list to a variable.
(same for the shorter list a few lines below)